components/esp_lvgl_port/CMakeLists.txt

```diff
@@ -1,29 +1,64 @@
-file(GLOB_RECURSE IMAGE_SOURCES images/*.c)
 
-idf_component_register(SRCS "esp_lvgl_port.c" ${IMAGE_SOURCES} INCLUDE_DIRS "include" REQUIRES "esp_lcd" PRIV_REQUIRES "esp_timer")
+#Get LVGL version
+idf_component_get_property(lvgl_ver lvgl__lvgl COMPONENT_VERSION)
+if(lvgl_ver EQUAL "")
+    idf_component_get_property(lvgl_ver lvgl COMPONENT_VERSION)
+endif()
+message(STATUS "LVGL version: ${lvgl_ver}")
+
+#Select folder by LVGL version
+if(lvgl_ver LESS "9.0.0")
```

Comment:
Should be `VERSION_LESS`, I think. Otherwise when the version 10.0.0 comes out, it will evaluate as `LESS` than 9.0.0 (lexicographically).

Reply:
@igrr Thank you for the warning :-) I fixed it.